code-challenges.js

```diff
@@ -9,9 +9,22 @@ var people = [
   { name: "arthur dent", occupation: "radio employee" }
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
+const sentenceMaker = (array) => {
+  // use .map to go through each element in the array
+  return array.map(value => {
+    // convert name to array so we can go through each element
+    let tempName = value.name.split(" ")
+    // declare a temp array variable for the cap words, then go through and capitalize the names
+    let capNameArray = []
+    tempName.map(value => {
+      capNameArray.push(value.charAt(0).toUpperCase() + value.substr(1))
+    })
+    // now put the sentence together and return it
+    return capNameArray.join(" ").concat(" is a ", value.occupation, ".")
+  })
+}
 
```

Comment:
As we talked about in class, this solution looks great, but the .push is just an extra step that could be removed with a refactor.